styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+box-sizing: border-box;
+}  
+
+body {
+margin: 0;
+background-color: #95CDED;
+color: #ffffff;
+display: flex;
+}
+
+aside {
+	width: 35%;
+	
+}
+/*
+img {
+	max-height: 500px;
+	max-width: 500px;
+}
+*/
+
+nav {
+	background-color: #95CDED;
```

Comment:
this isn't necessary as you have set the background-color and color on the parent element body